airflow/api_connexion/endpoints/dag_run_endpoint.py

```diff
@@ -172,7 +172,7 @@ def _fetch_dag_runs(
         query = query.where(DagRun.updated_at <= updated_at_lte)
 
     total_entries = get_query_count(query, session=session)
-    to_replace = {"dag_run_id": "run_id"}
+    to_replace = {"dag_run_id": "run_id", "execution_date": "logical_date"}
```

Comment:
This workaround is due to `apply_sorting` using the _raw SQL column name_, not the SQLAlchemy definition, so we need to explicitly fix the name separately. Eventually we should rename the argument on this API to `logical_date` instead.